### ISSUE TITLE
Error parsing `/etc/os-release` when values contain "=" symbols.

### ISSUE THREAD

**Issue by hasanmerkit** [PJ-Singh-001/Cubic#196]
**When equal letter (=) in website url string like this: (/etc/os-release)**
![image](https://user-images.githubusercontent.com/18331715/232206708-18ee02f2-1659-4d07-acc6-25f64b33907e.png)

Stuck at:
![image](https://user-images.githubusercontent.com/18331715/232206803-4561e8b2-dc01-43d8-9632-a127698782e8.png)

Terminal return:
```
Exception in thread Thread-67 (navigate):
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/share/cubic/cubic/navigator.py", line 613, in navigate
    result = new_page.enter(action, page) if new_page else None
  File "/usr/share/cubic/cubic/pages/generate_page.py", line 146, in enter
    is_error = copy_kernel_files()
  File "/usr/share/cubic/cubic/pages/generate_page.py", line 379, in copy_kernel_files
    is_pop_os_based = constructor.os_is_distribution('pop', model.project.custom_root_directory)
  File "/usr/share/cubic/cubic/utilities/constructor.py", line 245, in os_is_distribution
    key, value = line.split('=')
ValueError: too many values to unpack (expected 2)
```
**When equal letter (=) removed:**
![image](https://user-images.githubusercontent.com/18331715/232206901-cec116e7-f971-45be-a558-fbf703e9dbed.png)

Successfuly.
![image](https://user-images.githubusercontent.com/18331715/232207128-aa2be71b-1cc5-404c-8dcf-e330e8920abe.png)

I think Cubic cannot parse "=" in /etc/os-release.

**Comment by PJ-Singh-001** [comment]
Thanks for reporting the bug and including detailed information.
I'll fix this asap!

**Comment by PJ-Singh-001** [comment]
Fix released in Cubic version 2023.04-81.
Please test and report any issues.

**Comment by hasanmerkit** [comment]
Works perfect <3 Thanks :)
